### PR TITLE
Feature/open due times

### PR DESCRIPTION
**Pull request by pandafulmanda**
Dependent on #1055 and openstax/tutor-server#1077

screen shot 2016-05-17 at 7 11 26 pm

screen shot 2016-05-18 at 3 00 13 pm

### CODE REVIEW

**Comment by pandafulmanda** [comment]
@philschatz this is ready for review, needs to be merged with openstax/tutor-server#1077

File: src/components/course-settings/set-timezone.cjsx [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+    {courseTimezone} = @state
+
+    timezonesToPick = _.map timezones, (timezone) =>
+      <TutorRadio
+        id={S.dasherize(timezone)}
+        value={timezone}
+        name={name}
+        checked={timezone is courseTimezone}
+        onChange={@onChange}/>
+
```
Comment (nathanstitt) [inline src/components/course-settings/set-timezone.cjsx]: missing 🔑 property

**Comment by nathanstitt** [comment]
Looks good to me @pandafulmanda.  Tested locally with the accompanying BE PR and seems to work correctly

File: src/components/course-settings/set-timezone.cjsx (hunk 1 above)
Comment (pandafulmanda) [inline src/components/course-settings/set-timezone.cjsx]: nice catch!  made an update

**Pull request merged**